common/proto/evm.token.proto

```diff
@@ -0,0 +1,45 @@
+syntax = "proto3";
+
+package messari.evm.token.v1;
+
+message Tokens {
+  repeated Token items = 1;
+}
+
+message Token {
+  string address = 1;
+  string name = 2;
+  string symbol = 3;
+  uint64 decimals = 4;
+  optional string total_supply = 5;
+}
+
+message Transfers {
+  repeated Transfer items = 1;
+}
+
+message Transfer {
+  string tx_hash = 1;
+  optional uint32 log_index = 2;
```

Comment:
why are `log_index` and `log_ordinal` optional? what are some scenarios these will be missing?

Reply:
I made them optional when I was going to make one proto for all chains, Updating that